### PR TITLE
AB#13515 Mapped fields that are missing from the Manage Columns dropdown

### PR DESCRIPTION
**Pull request by don-aot**

### CODE REVIEW

File: applications/Unity.GrantManager/src/Unity.GrantManager.Application/GrantApplications/GrantApplicationAppService.cs [hunk 1]
```diff
@@ -141,6 +146,12 @@ select new
             appDto.OrganizationType = grouping.First().applicant?.OrganizationType ?? string.Empty;
             appDto.Assignees = BuildApplicationAssignees(grouping.Select(s => s.applicationUserAssignment).Where(e => e != null), grouping.Select(s => s.applicationPerson).Where(e => e != null)).ToList();
             appDto.SubStatusDisplayValue = MapSubstatusDisplayValue(appDto.SubStatus);
+            appDto.DeclineRational = MapDeclineRationalDisplayValue(appDto.DeclineRational);
+            appDto.ContactFullName = grouping.First().applicantAgent?.Name ?? string.Empty; ;
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Application/GrantApplications/GrantApplicationAppService.cs:150]: @don-aot  - double semi colons can be cleanup up please

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js [hunk 2]
```diff
@@ -776,11 +788,101 @@
             data: 'organizationName',
             className: 'data-table-header',
             render: function (data) {
-                return data ?? '';
+                return data || '{Organization Name}';
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js:791]: why swap to falsey? i think for these fields and the fields below nullish (??) would make more sense.

File: applications/Unity.GrantManager/src/Unity.GrantManager.Application/GrantApplications/GrantApplicationAppService.cs (hunk 1 above)
Comment (don-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Application/GrantApplications/GrantApplicationAppService.cs]: @AndreGAot  Updated;

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js (hunk 2 above)
Comment (don-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js]: @AndreGAot  Updated;

**Review by AndreGAot — APPROVED** [review]
Ready for Dev

**Pull request merged**